editor/configs/map.ts

```diff
@@ -18,15 +18,20 @@ export const MAP_MAX_BOUNDS: [[number, number], [number, number]] = [
   [-96, 32],
 ];
 
+/** Different basemap style options */
+export const mapStyleOptions = {
+  darkStreets: "mapbox://styles/mapbox/dark-v11",
+  lightStreets: "mapbox://styles/mapbox/streets-v12",
```

Comment:
this is another option for light mode but it was looking very white, what do yall think?

<img width="2233" height="1205" alt="Screenshot from 2025-08-28 15-06-57" src="https://github.com/user-attachments/assets/8c0d93f5-0110-4b70-9d08-485a9653b085" />
 <img width="2233" height="1205" alt="Screenshot from 2025-08-28 15-07-14" src="https://github.com/user-attachments/assets/bcdad71e-1c73-403a-b3e6-acfd4456dd72" />

Reply:
VS what i currently have in this PR:

<img width="2233" height="1205" alt="Screenshot from 2025-08-28 15-12-07" src="https://github.com/user-attachments/assets/6c83a9d6-72f9-402e-a709-46a42bf03022" />
![Uploading Screenshot from 2025-08-28 15-12-13.png…]()

Reply:
I like the full color street map in light mode! I think the `light` map could be useful to draw more contrast between the map and the crash points, esp for seeing clusters, but I think `streets` is the best default and has tons of utility 🚢

Reply:
I keep going back and forth on streets versus `light`. I like the full-color `streets` because I find it helpful for wayfinding.

I had initially conceived of three choices:
- **Analysis** - `light` or `dark` (default)
- **Streets** - `streets`
- **Aerial**

But Rose pointed out that **Analysis** is confusing and, echoing Tilly, it's better to keep the options simple. 

I'm open to going with `light` instead of `streets` though. In any case, I will open an issue to fix the poor contrast on streets and especially with aerials. We might want to ship that as a patch, even.